docs/data-sources/virtual_host_names.md

```diff
@@ -0,0 +1,24 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "pingfederate_virtual_host_names Data Source - terraform-provider-pingfederate"
+subcategory: ""
+description: |-
+  Manages VirtualHostNames.
+---
+
+# pingfederate_virtual_host_names (Data Source)
```

Comment:
The example file needs to be named "data-source.tf" so that it shows up in this generated doc

Reply:
done